http/client.go

```diff
@@ -37,18 +37,6 @@ var DefaultTransport = &http.Transport{
 	IdleConnTimeout:     30 * time.Second,
 }
 
-// DefaultClient is a dp-net specific http client with sensible timeouts,
-// exponential backoff, and a contextual dialer.
-var DefaultClient = &Client{
```

Comment:
Can we not have this variable DefaultClient but not point to Client?

So `var DefaultClient = Client{ ...`

Reply:
[Discussed in person] and not sure if a cleaner way to do this without overwriting the default ransport.